src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -194,10 +194,7 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
 
     private _showInfoIcon = (item: PortfolioPlanningQueryResultItem, isPredecessor: boolean): boolean => {
         // only show info icon if the item is in InProgress state.
-        const states = this.state.workItemTypeMappedStates.filter(
-            item => item.workItemTypeName === item.workItemTypeName
-        )[0].states;
-        const statesForInProgress = Object.keys(states).filter(key => states[key] === "InProgress");
+        const statesForInProgress = this.state.workItemTypeMappedStatesInProgress[item.WorkItemType.toLowerCase()];
         if (statesForInProgress.indexOf(item.State) === -1) return false;

```

Comment:
nit: You should protect against discrepancies in state name casing... unlikely since the work item state is coming from Odata service with the same casing as the work item states catalog in project backlog configuration...